src/Compilers/VisualBasic/Portable/Operations/VisualBasicOperationFactory.vb

```diff
@@ -1024,14 +1011,7 @@ Namespace Microsoft.CodeAnalysis.Operations
             Dim constantValue As [Optional](Of Object) = New [Optional](Of Object)()
             Dim isImplicit As Boolean = boundCaseBlock.WasCompilerGenerated
 
-            ' Some bound nodes used by the boundCaseBlock.CaseStatement.CaseClauses are also going to be used in boundCaseBlock.CaseStatement.ConditionOpt.
-            ' If we simply create another tree based on boundCaseBlock.CaseStatement.ConditionOpt, due to the caching we will end up with the same
-            ' IOperation nodes in two trees, and two parents will compete for assigning itself as the parent - trouble. To avoid that, we simply use
-            ' a new factory to create IOperation tree for the condition. Note that the condition tree is internal and regular consumers cannot get to
```

Comment:
>BitVector.Null [](start = 87, length = 14)

Why ```BitVector.Null``` is the right value here, aren't all arguments default? Since the entire access is compiler generated and doesn't correspond to anything in the syntax, that is probably fine. Consider adding comment. However, there is a question, why we are not doing the same for GetEnumeratorMethod and MoveNextMethod?

Reply:
Note: this comment was for line 1192

---
In reply to: [190031533](https://github.com/dotnet/roslyn/pull/26781#discussion_r190031533) [](ancestors = 190031533)

Reply:
We should be passing DefaultArguments through here, will fix.

---
In reply to: [190075484](https://github.com/dotnet/roslyn/pull/26781#discussion_r190075484) [](ancestors = 190075484,190031533)